src/Components/WebAssembly/DevServer/src/Server/Startup.cs

```diff
@@ -29,6 +29,21 @@ public static void Configure(IApplicationBuilder app, IConfiguration configurati
 
         app.UseWebAssemblyDebugging();
 
+        app.Use(async (ctx, next) =>
```

Comment:
Can the DevServer find out about the project build properties somehow?  I don't think we want COP for arbitrary blazor projects.  it could break someone's existing single threaded project, I think?

Reply:
The last change works with the property is put in a csproj

Reply:
@radical Do you know if there is a way to "evaluate" properties passed as command-line arguments during dotnet run? The passed property has correct value during build, but when RunArguments evaluate, it ignores value from command-line